prototypes/recipes/korlex/recipes-korlex.lua

```diff
@@ -56,8 +56,8 @@ RECIPE {
         {type = "item",  name = "seaweed",      amount = 10},
         {type = "item",  name = "salt",         amount = 10},
         {type = "item",  name = "fawogae",      amount = 10},
-        {type = "fluid", name = "steam",        amount = 100},
         {type = "fluid", name = "fish-oil",     amount = 100},
+        {type = "fluid", name = "steam",        amount = 100},
```

Comment:
Please instead use the fluidbox_index property here.

`type = "fluid", name = "steam",        amount = 100, fluidbox_index = 2},`